### PR TITLE
Update Uninstalling_a_DataMiner_Agent.md

### PR DESCRIPTION
**Pull request by brmdias**
Adding procedure to use the uninstaller from DataMiner. This is the most common procedure used.

### CODE REVIEW

File: user-guide/Advanced_Functionality/DataMiner_Agents/Uninstalling_a_DataMiner_Agent.md [hunk 1]
```diff
@@ -4,72 +4,98 @@ uid: Uninstalling_a_DataMiner_Agent
...
 
-You can uninstall the DataMiner Agent software as follows:
+There are two possibilities to uninstall a DataMiner Agent:
+
+- [Using the DataMiner Installer](#uninstalling-a-dma-using-the-dataminer-installer)
+- [Using a manual procedure](#uninstalling-a-dma-using-a-manual-procedure))
+
+## Uninstalling a DMA using the DataMiner Installer
+
+1. Download the DataMiner installer from [DataMiner Dojo](https://community.dataminer.services/download/dataminer-installer-v10-2).
```
Comment (MariekeGO) [inline user-guide/Advanced_Functionality/DataMiner_Agents/Uninstalling_a_DataMiner_Agent.md:14]: If people installed DataMiner with the 10.0 installer or an older installer, will this procedure work? Or will this only work if they installed DataMiner using the 10.2 installer?

Comment (MariekeGO) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Uninstalling_a_DataMiner_Agent.md:14]: Quick update: I've been informed that this still needs to be investigated and tested, so for now we can't merge this yet.

Comment (klaasdebeuf) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Uninstalling_a_DataMiner_Agent.md:14]: https://collaboration.dataminer.services/task/210483

Comment (klaasdebeuf) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Uninstalling_a_DataMiner_Agent.md:14]: It works

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**